corehq/apps/cloudcare/views.py

```diff
@@ -263,6 +263,18 @@ def get_cases(request, domain):
     ids_only = string_to_boolean(request.REQUEST.get("ids_only", "false"))
     case_id = request.REQUEST.get("case_id", "")
     footprint = string_to_boolean(request.REQUEST.get("footprint", "false"))
+
+    if toggles.HSPH_HACK.enabled(domain):
+        hsph_case_id = request.REQUEST.get('hsph_hack', None)
+        if hsph_case_id != 'None' and hsph_case_id:
```

Comment:
yea unfortunately the current query manager we use in touchforms doesn't properly encode the parameters. that's why we have a bunch of `string_to_boolean` type functions floating around. probably should fix that